src/DependencyInjection/PimcoreEcommerceFrameworkExtension.php

```diff
@@ -597,7 +597,7 @@ private function registerOfferToolConfig(ContainerBuilder $container, array $con
 
         $container->setParameter(
             'pimcore_ecommerce.offer_tool.order_storage.parent_folder_path',
-            $config['order_storage']['parent_folder_path']
+            $config['order_storage']['offer_parent_path'] ?? $config['order_storage']['parent_folder_path']
```

Comment:
is it a little too hackish?
It's because of https://github.com/pimcore/ecommerce-framework-bundle/blob/4c2f9f5f20d128a1f61ef0a5335516c0a8cba1c5/src/Resources/config/offer_tool.yaml#L11-L15
any suggestions?

Reply:
I see - I would say this is fine!